.travis.yml

```diff
@@ -11,7 +11,7 @@ env:
 
 before_script:
   - mix do deps.get, compile --warnings-as-errors, coveralls.json
-  - MIX_ENV=dev travis_wait 20 mix dialyzer --plt
+  - MIX_ENV=dev travis_wait 30 mix dialyzer --plt
```

Comment:
😬

Reply:
travis has been timing out running dialyzer, as the plt cache is invalidated with the elixir ver bump